tables/iRO/official/items.txt

```diff
@@ -8556,3 +8555,4 @@
 20763#Costume_Amistr_Bag#
 31052#Costume_Alchemist_Mask#
 23164#Granpa_Speed_potion#
+20164#Costume:_Duneyrr_Helm#
```

Comment:
Yep. ;)

I know this is a joke, but I have to respond anyway, because I'm OCD like that:

Yes, if anybody had the book, but in practice they don't do that for items which were instantiated in the past and still exist on the server. "Rental Saharic Book" might be a good choice for replacement simply due to it being a rental (because the rental expiration wouldn't change, so even if somebody now had a rental Duneyrr Helm, it would disappear at the scheduled time anyway). In this particular case they've never introduced any of the Rental Saharic items, so there's no in-game impact.